### PR TITLE
Enhance the output of the Perun

### PR DESCRIPTION
**Pull request by tfiedor**
This PR:

1. Enhances the output on the commandline,
2. Remodels the way information is given to user,
3. Extract styling and colours to isolate package.

### CODE REVIEW

**Comment by tfiedor** [comment]
Preview of the new look:
output-rm
output-add
output-init
output-collect
output-fuzzing

**Review by tfiedor — COMMENTED** [review]
I reviewed my code, and found few errors, which I will fix in next commit.

File: perun/fuzz/evaluate/by_perun.py [hunk 1]
```diff
@@ -31,9 +31,11 @@ def baseline_testing(
     postprocessor: list[str],
     minor_version_list: list[MinorVersion],
     **kwargs: Any,
-) -> Iterable[tuple[CollectStatus, Profile, str]]:
+) -> Iterable[tuple[CollectStatus, Profile, Job]]:
     """Generates a profile for specified command with init seeds, compares each other.
 
+    TODO: This might need some checking and tweaking as I believe it is quite shady
```
Comment (JiriPavela) [inline perun/fuzz/evaluate/by_perun.py:37]: What's shady about this? Is it something serious? If so, we should probably open a new issue for it so we don't forget about it.

**Review by JiriPavela — APPROVED** [review]
Great, I like the new logging API and the way you can build the terminal output. I noticed the log messages regarding job progress in one of the screenshots, should we perhaps consider enhancing logging with some terminal progress bar (e.g., think of apt or dnf package managers)?

**Comment by tfiedor** [comment]
> Great, I like the new logging API and the way you can build the terminal output. I noticed the log messages regarding job progress in one of the screenshots, should we perhaps consider enhancing logging with some terminal progress bar (e.g., think of apt or dnf package managers)?

Good point, though not for the particular case, I can see the usage for the "ticks". I will merge this and make an issue out of this.

**Pull request merged**